src/tools/miri/src/shims/windows/foreign_items.rs

```diff
@@ -203,6 +203,16 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
                 )?;
             }
 
+            // Number of cores, but more accurate vaulue, than GetSystemInfo.dwNumberOfProcessors
+            // if number of cores > 64. If called with ALL_PROCESSOR_GROUPS as groupnumber,
+            // returns total number of cores, instead of number in group, see
+            // https://learn.microsoft.com/en-us/windows/win32/api/winbase/nf-winbase-getactiveprocessorcount
```

Comment:
This sounds like we should do `throw_unsup_format!` if the groupnumber is *not* ALL_PROCESSOR_GROUPS, since we are not correctly handling that case.

Reply:
`vaulue` typo.